clang/test/CodeGenSYCL/special-classes.cpp

```diff
@@ -0,0 +1,103 @@
+// RUN: %clang_cc1 -S -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -emit-llvm %s -o -  | FileCheck %s
+
```

Comment:
Do we need this test? Is this checking anything additional to what we already check in existing accessor/sampler/stream tests? If we're keeping test, please add a comment describing test